package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -177,13 +178,43 @@ class H5MDReader(base.ReaderBase):
              'force': None}
     # Translate H5MD units (https://nongnu.org/h5md/modules/units.html) to MDAnalysis units.
     _unit_translation = {
+        'time': {
+            'ps': 'ps',
+            'fs': 'fs',
+            'ns': 'ns',
+            'second': 'second',
```

Comment:
It looks more obvious when the right hand sides are consistent, e.g., all “s”

Reply:
DId you check if MDA can actually work with the units that you’re converting? Have a look into MDAnalysis.lib.units.

Reply:
Yep, that's how I filled the dictionary. I just went through `timeUnit_factor`, `lengthUnit_factor`, etc and filled in what the H5MD equivalent is.

Reply:
Ok, great, good that you looked it up while i was just pontificating.

Reply:
What did you mean by making the right hand sides consistent?

Reply:
instead of second, sec, s just have whatever MDAnalysis uses... if it uses all three, pick one eg seconds.